CHANGELOG.md

```diff
@@ -2,6 +2,112 @@ TODO: This document was manually maintained so might be incomplete. The
 automation effort is tracked in
 https://github.com/kubernetes/client-go/issues/234.
 
+# v6.0.0
+
+* `core/v1.Taint.TimeAdded` became a pointer
+
+   * [https://github.com/kubernetes/kubernetes/pull/43016](https://github.com/kubernetes/kubernetes/pull/43016)
+
+* Certificate manager was moved to a shareable location
+
+   * [https://github.com/kubernetes/kubernetes/pull/49654](https://github.com/kubernetes/kubernetes/pull/49654)
+
+* `unstructued.Unstructured` got getters and setters
+
+    * [https://github.com/kubernetes/kubernetes/pull/51940](https://github.com/kubernetes/kubernetes/pull/51940)
+
+* Redirect behavior is restored for proxy subresources
+
+    * [https://github.com/kubernetes/kubernetes/pull/52933](https://github.com/kubernetes/kubernetes/pull/52933)
+
+* Swagger 1.2 support was removed from kubectl
+
+    * [https://github.com/kubernetes/kubernetes/pull/53441](https://github.com/kubernetes/kubernetes/pull/53441)
+
+* Unset `creationTimestamp` field is output as `null`
+
+    * [https://github.com/kubernetes/kubernetes/pull/53464](https://github.com/kubernetes/kubernetes/pull/53464)
+
+* `ObjectCopier` interface was removed (requires switch to new generators with DeepCopy methods)
+
+    * [https://github.com/kubernetes/kubernetes/pull/53525](https://github.com/kubernetes/kubernetes/pull/53525)
+
+* Workloads api types is moved into new apps/v1 version
+
+    * [https://github.com/kubernetes/kubernetes/pull/53679](https://github.com/kubernetes/kubernetes/pull/53679)
+
+* Random string generation functions are optimised
+
+    * [https://github.com/kubernetes/kubernetes/pull/53720](https://github.com/kubernetes/kubernetes/pull/53720)
+
+*  */subresource specification is allowed in RBAC roles
+
+    * [https://github.com/kubernetes/kubernetes/pull/53722](https://github.com/kubernetes/kubernetes/pull/53722)
+
+* Scaling of resources in arbitrary API groups is allowed
```

Comment:
Introduces a polymorphic scale client, which supports scaling of resources in arbitrary API groups.

New feature.